persist/fs/commitlog/info.go

```diff
@@ -49,7 +50,7 @@ func ReadLogInfo(filePath string) (start time.Time, duration time.Duration, inde
 		return time.Time{}, 0, 0, err
 	}
 	logDecoder := msgpack.NewDecoder(nil)
-	logDecoder.Reset(bytes)
+	logDecoder.Reset(encoding.NewDecoderStream(bytes))
```

Comment:
Should we pool Decoder/Reader/DecoderStream here?

Reply:
This is only O(files) of commit log files, not O(entries).  The amount of files should be less than a hundred (worst-ish case) so this is fine, `ReadLogInfo(...)` is not in the hot path.